Documentation/contributing/development/contributing_guide.rst

```diff
@@ -14,29 +14,31 @@ Clone and Provision Environment
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 #. Make sure you have a `GitHub account <https://github.com/join>`_
-#. Clone the cilium repository into your ``GOPATH``.
+#. Fork the Cilium repository to your GitHub user or organization.
+#. Clone your Cilium fork into your ``GOPATH``, and setup the base repository as ``upstream`` remote:
 
    ::
 
-      mkdir -p $GOPATH/src/github.com/cilium
-      cd $GOPATH/src/github.com/cilium
-      git clone https://github.com/cilium/cilium.git
+      mkdir -p ${GOPATH}/src/github.com/${YOUR_GITHUB_USERNAME_OR_ORG}
+      cd ${GOPATH}/src/github.com/${YOUR_GITHUB_USERNAME_OR_ORG}
```

Comment:
Considering it's a fork, I was not sure if these should be left to:
```suggestion
      mkdir -p ${GOPATH}/src/github.com/cilium
      cd ${GOPATH}/src/github.com/cilium
```

Reply:
they should be left to cilium.

Reply:
OK, changing that back.